customtypes/accordionitem/index.json

```diff
@@ -7,31 +7,21 @@
     "Main": {
       "title": {
         "type": "Text",
-        "config": {
-          "label": "Title",
-          "placeholder": ""
-        }
+        "config": { "label": "Title", "placeholder": "" }
       },
       "slices": {
         "type": "Slices",
         "fieldset": "Slice Zone",
         "config": {
           "choices": {
-            "button_slice": {
-              "type": "SharedSlice"
-            },
-            "embed_slice": {
-              "type": "SharedSlice"
-            },
-            "image_slice": {
-              "type": "SharedSlice"
-            },
-            "text_slice": {
-              "type": "SharedSlice"
-            }
```

Comment:
This is a general comment for the whole PR.
It's usually preferable, when possible, to avoid formatting changes (unless you are fixing linter errors or updating to match the code style guidelines). 
Pure format or layout changes tend to clutter the PR and make it harder for folks to distinguish or find the (functional) changes to the code and make it more difficult to compare the new and old code.  There's also a risk of introducing  inconsistent layouts across or within files.

Reply:
Yeah , I think the slice machine kept changing the files.

Reply:
all the changes that are deleted lines and written lines in the customtype jsons were not done by me.

Reply:
Ugh - that's annoying. I wonder if it's an issue with (different) SliceMachine versions.